heksher/api/v1/settings.py

```diff
@@ -59,8 +59,8 @@ async def declare_setting(input: DeclareSettingInput, app: HeksherApp = applicat
     Ensure that a setting exists, creating it if necessary.
     """
     new_setting = input.to_setting()
-    existing = await app.db_logic.get_setting(input.name)
```

Comment:
no need to postface the name of every variable with its type. In a method called "declare setting", `existing` is obviously a setting.